libheif/api/libheif/heif.h

```diff
@@ -251,6 +251,9 @@ enum heif_suberror_code
 
   heif_suberror_No_avcC_box = 143,
 
+  // we got a mini box, but could not read it properly
+  heif_suberror_No_mini_box = 149,
```

Comment:
Maybe we should rename the error to something else so that it fits the semantics?
Something like `heif_suberror_Invalid_mini_box` ?

Reply:
Fixed.